buildspec.yml

```diff
@@ -4,23 +4,17 @@ phases:
 
   install:
     runtime-versions:
-      java: corretto11
-
+      java: corretto17
     commands:
-#
-#      # Upgrade AWS CLI to the latest version
-#      - pip3 install --upgrade awscli
-#
-      # Install aws-sam-cli
-      - pip3 install aws-sam-cli
```

Comment:
we dont need this anymore?

Reply:
This is why the karate test is failing I think.

this needs to be changed to - pip3 install aws-sam-cli -U